frontend/src/login/Login.tsx

```diff
@@ -0,0 +1,65 @@
+import {FormEvent, useState} from "react";
+import {useNavigate} from "react-router-dom";
+
+
+const Login = () => {
+
+    const [emailLogin, setEmailLogin] = useState('')
+    const [password, setPassword] = useState('')
+    const [errorMessage, setErrorMessage] = useState('')
+    const navigate = useNavigate()
+
+
+    const login = (event : FormEvent<HTMLFormElement>) => {
+        event.preventDefault()
+        fetch(`${process.env.REACT_APP_BASE_URL}/api/login`, {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify({
+                "emailLogin": emailLogin,
+                "password": password
+            })
+        })
+            .then(response => {
+                if (!response.ok) {
+                    setErrorMessage("Email or password not valid")
+                    throw new Error()
```

Comment:
Ich würde das `setErrorMessage("Email or password not valid")` weiter unten nach dem letzten then im .catch() machen. So ganz ohne catch ist es komisch, dass du throw new Error machst - das muss ja irgendwo gefangen werden, wie in Java mit try..catch..

Reply:
ok, das stimmt